packages/core/src/context/content-audit.ts

```
@@ -0,0 +1,123 @@
+        uiResourceParts: 0,
+        textLength: 0,
+    };
+}
+
+async function sha256Hex(input: string): Promise<string | undefined> {
+    const subtle = globalThis.crypto?.subtle;
+    if (subtle === undefined) {
+        return undefined;
+    }
+
+    const digest = await subtle.digest('SHA-256', new TextEncoder().encode(input));
```

ESLint

[error] 'TextEncoder' is not defined.

(no-undef)